src/devices/acurite.c

```diff
@@ -408,13 +412,77 @@ similar RF encoding and data format:
 - 592TXR temperature and humidity sensor
 - 5-n-1 weather station
 - 6045M Lightning Detector with Temperature and Humidity
+- Atlas
 
     CC RR IIII | IIII IIII | pBMMMMMM | pxxWWWWW | pWWWTTTT | pTTTTTTT | pSSSSSSS
     C:2d R:2d ID:12d 1x BATT:1b TYPE:6h 1x ?2b W:5b 1x 3b T:4b 1x 7b S: 1x 7d
 
 @todo - refactor, move 5n1 and txr decoding into separate functions.
 @todo - TBD Are parity and checksum the same across these devices?
         (opportunity to DRY-up and simplify?)
+
```

Comment:
This callback is getting pretty long.  I suggest breaking out both the Atlas specific code and the comments that document the function into a separate function, similar to the 6045

Reply:
I agree, this would improve readability quite a bit.